src/main/backend.ts

```diff
@@ -150,30 +142,95 @@ export class MainBackend extends BaseBackend<ElectronOptions> implements CommonB
     const hubScope = getCurrentHub().getScope();
     if (hubScope) {
       hubScope.addScopeListener(updatedScope => {
-        const cloned = Scope.clone(updatedScope);
+        const scope = Scope.clone(updatedScope);
         // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-        (cloned as any)._eventProcessors = [];
+        (scope as any)._eventProcessors = [];
         // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-        (cloned as any)._scopeListeners = [];
+        (scope as any)._scopeListeners = [];
+
         // if we use the crashpad minidump uploader we have to set extra whenever the scope updates
         if (this._options.useCrashpadMinidumpUploader === true) {
-          getCurrentHub().captureEvent(
-            {
-              // @ts-ignore __INTERNAL_MINIDUMP is not assignable to event
-              __INTERNAL_MINIDUMP: true,
-            },
-            {
-              data: {
-                __sentry__: true,
-              },
-            },
+          forget(
+            this._getNativeUploaderEvent(scope).then(event => {
+              // Update the extra parameters in the main process
+              const mainParams = this._getNativeUploaderExtraParams(event);
+              for (const key of Object.keys(mainParams)) {
+                crashReporter.addExtraParameter(key, mainParams[key]);
+              }
+
+              // Modify the event to point to renderer crash
+              event.extra = { ...event.extra, crashed_process: 'renderer' };
+              // Update the extra parameters in every renderer process
+              const rendererParams = this._getNativeUploaderExtraParams(event);
+              for (const window of BrowserWindow.getAllWindows()) {
+                if (!window.webContents.isDestroyed()) {
+                  window.webContents.send(IPC.EXTRA_PARAMS, rendererParams);
```

Comment:
Do we absolutely need to do this broadcast? It looks like `initialScope` covers the most important case to have shared scope between _all_ processes.

Reply:
If you want breadcrumbs and full context with native renderer crashes via the Electron uploader, this is the only way to do that.

My understanding was that `initialScope`/`globalExtra` was to add minimal context for GPU crashes which are not covered by the above.

Reply:
If `initialScope` is enough context for ALL native crash reports via the Electron uploader, we can pull this PR entirely.

@HazAT added usage of `addExtraParameter` a year ago [here](https://github.com/getsentry/sentry-electron/pull/244/files#diff-39928ad64d67bbb28adaeb7b5e0b6580af610e5b19d5a3cf730d59c39c7bffa1R118) so I had assumed full context was a requirement. There are a couple of issues with the previous implementation:
- `addExtraParameter` called in `sendEvent` so context would only get added after an event
- `addExtraParameter` needs calling in the renderer if you want context there